conf/repos.yaml

```diff
@@ -0,0 +1,11 @@
+REPOS:
+  CAPSULE_REPO: ""
+  RHEL6_OS: ""
+  RHEL7_OS: ""
+  SATTOOLS_REPO:     # TOOLS_URL_RHEL6, TOOLS_URL_RHEL7, TOOLS_URL_RHEL8
+    RHEL6: ""
+    RHEL7: ""
+    RHEL8: ""
+  SATMAINTENANCE_REPO: ""
+  SATELLITE6_REPO: ""
+  PUPPET4_REPO: ""
```

Comment:
@devendra104 do we need puppet repo for upgrades?

Reply:
puppet repo is not required for upgrade, it is required for puppet version upgrade.